src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+
+    private double trapezoidHeight;
+    private String color;
+    private String figure;
+    private double sideA;
+    private double sideB;
```

Comment:
```suggestion
    private String color;
    private FigureType figureType;
```

we have only two common states for all figures - color and FigureType